project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
I meant that either to make it all in one repo as sub-projects or to have separate repos and manage backend as an artifact. If I said otherwise I must have misunderstood what the context was. 

So just to confirm, you're still pro-separate-repo, right? If so I'll get it set up ASAP